.github/workflows/build.yml

```
@@ -132,6 +132,8 @@ jobs:
           # Some of the above are executed by root, creating ~/.cargo/git as
           # that user, blocking downloads of own libraries.
           rm -rf ~/.cargo
+          # Something left directories that break things
+          (cd RIOT && git clean -fxdv)
```

Looks like the CI error is caused by the `-v` flag here:

```suggestion
          (cd RIOT && git clean -fxd)
```